charts/kuberpult/values.yaml

```diff
@@ -91,6 +91,8 @@ frontend:
 # See frontend-service.yaml for automatically added annotations.
   service:
     annotations: {}
+    batchClient:
```

Comment:
This place is slightly wrong because it contains the annotations of the kubernetes service. Rather move it one level up where also `maxWaitDuration` is.

Reply:
Also add a documentation that this must be higher than the cd servers gitNetworkTimeout